Backpack-SwiftUI/SearchInputSummary/Classes/BPKSearchInputSummary.swift

```diff
@@ -113,6 +122,8 @@ public struct BPKSearchInputSummary: View {
         case .icon(let icon):
             BPKIconView(icon)
                 .foregroundColor(.textPrimaryColor)
+        case .none:
+            EmptyView()
```

Comment:
lets write this prefixView in a different way to avoid having to use EmptyView:
```swift
@ViewBuilder
private var prefixView: some View {
  if let inputPrefix {
    switch inputPrefix {
      case .text(let prefixText):
          BPKText(prefixText, style: .bodyDefault)
              .foregroundColor(.textSecondaryColor)
      case .icon(let icon):
          BPKIconView(icon)
              .foregroundColor(.textPrimaryColor)
    }
  }
}

Reply:
Good catch! thanks